tooling/template-base/components/Navigation/CatalogMobileNavBar.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { useState } from 'react';
+import Logo from './Logo';
+import avatar from '../../renderer/avatar.png';
+import xicon from '../../renderer/xicon.png';
+
+export default function MobileNavBar() {
+  const [navbar, setNavbar] = useState(false);
+
+  return (
+    <nav className="w-full bg-white shadow py-4">
+      <div className="justify-between px-4 md:items-center md:flex">
+        <div>
+          <div className="flex items-center justify-between md:block">
+            {/* logo */}
+            <div className="">
+              <Logo />
+            </div>
+            <div className="md:hidden">
+              <button
+                className="p-2 text-gray-700 rounded-md outline-none focus:border-gray-400 focus:border"
+                onClick={() => setNavbar(!navbar)}
+              >
+                {navbar ? (
+                  <button
+                    id="icon"
+                    type="submit"
+                    className="block h-9 w-9 md:hidden"
+                    onClick={() => setNavbar(!navbar)}
+                  >
+                    <img src={xicon} />
+                  </button>
+                ) : (
+                  <button
+                    id="icon"
+                    type="submit"
+                    className="block h-9 w-9 md:hidden"
+                    onClick={() => setNavbar(!navbar)}
+                  >
+                    <img src={avatar} />
```

Comment:
Users can upload their own profile pics, so it doesn't quite work to use a hardcoded image. If you pass the `CurrentUser` down or fetch it from context, it should have an `asset` property (provided the user has uploaded a picture). That's the asset we'll want to use in place of the hardcoded avatar image. 

In the event they don't have an image uploaded, we've defaulted to using the user's initials

https://github.com/thoughtindustries/ti/blob/5e738eddc579a0406e8a6fc1b4d759e48c17e96e/assets/javascripts/lms/templates/components/ti-avatar.hbs#L25

https://github.com/thoughtindustries/ti/blob/5e738eddc579a0406e8a6fc1b4d759e48c17e96e/assets/javascripts/lms/components/ti-avatar.js#L12

Reply:
is the `asset` property part of the `currentUser`? I can't seem to see it in `types.ts`